src/index.js

```diff
@@ -1 +1,87 @@
+/* eslint-disable no-console, no-console,max-len */
 'use strict';
+
+const http = require('http');
+const fs = require('fs');
+const { formidable } = require('formidable');
+
+const PORT = 3030;
+
+const server = http.createServer(async(req, res) => {
+  if (req.url === '/api/form' && req.method.toLowerCase() === 'post') {
+    const form = formidable();
+
+    let fields;
+
+    try {
+      [fields] = await form.parse(req);
+    } catch (err) {
+      res.writeHead(err.httpCode || 400, { 'Content-Type': 'text/plain' });
+      res.end(String(err));
+
+      return;
+    }
+
+    const expenseData = {
+      date: fields.date[0],
+      title: fields.title[0],
+      amount: fields.amount[0],
+    };
+
+    const parseToJson = JSON.stringify(expenseData);
+    const createPath = `src/expenses/history-${fields.date[0]}.txt`;
```

Comment:
You are storing the expense data in a '.txt' file rather than a '.json' file. If you are storing JSON data, the file extension should be '.json'.

Reply:
Not resolved